# Joint single-cell profiling resolves 5mC and 5hmC and reveals their distinct gene-regulatory functions

Emily B. Fabyanic$, Peng Hu$, Qi Qiu$, Kiara N. Berríos, Daniel R. Connolly, Tong Wang, Jennifer Flournoy, Zhaolan Zhou, Rahul M. Kohli, Hao Wu #

$ These authors contributed equally: E.F., P.H. and Q.Q.; 

#Correspondence should be addressed to H.W. (haowu2@pennmedicine.upenn.edu).


# Summary
Oxidative modification of 5-methylcytosine (5mC) by TET DNA dioxygenases generates 5-hydroxymethylcytosine (5hmC), the most abundant form of oxidized 5mC. Existing single-cell bisulfite sequencing methods cannot resolve  5mC and 5hmC, leaving the cell-type-specific regulatory mechanisms of TET and 5hmC largely unknown. Here we present Joint single-nucleus (hydroxy)methylcytosine sequencing (Joint-snhmC-seq), a scalable and quantitative approach that simultaneously profiles 5hmC and true 5mC in single cells by harnessing differential deaminase activity of APOBEC3A towards 5mC and chemically protected 5hmC. Joint-snhmC-seq profiling of single nuclei from the mouse brains reveals an unprecedented level of epigenetic heterogeneity of both 5hmC and true 5mC at single-cell resolution. We show that cell-type-specific profiles of 5hmC or true 5mC improve multi-modal single-cell data integration, enable accurate identification of neuronal subtypes, and uncover context-specific regulatory effects of cell-type-specific genes by TET enzymes.  

# Joint-snhmC-seq workflow
![Joint-snhmC-seq_diagram](https://github.com/wulabupenn/Joint-snhmC-seq/blob/main/Joint-snhmC-seq_diagram.png)

# Comparison between single and dual epigenetic modality sequencing in single cells
![Joint-snhmC-seq_diagram2](https://github.com/wulabupenn/Joint-snhmC-seq/blob/main/Joint-snhmC-seq_diagram2.png)

# Code
* Mapping pipelines
* Codes for plotting figures can be found [here]
(https://github.com/Xiaolong0803/Fabyanic-NATURE-BIOTECHNOLOGY-2023).

# Preprint
Quantitative single cell 5hmC sequencing reveals non-canonical gene regulation by non-CG hydroxymethylation

Emily B. Fabyanic*, Peng Hu*, Qi Qiu*, Tong Wang, Kiara N. Berríos, Jennifer Flournoy, Daniel R. Connolly, Zhaolan Zhou, Rahul M. Kohil, Hao Wu

doi: https://doi.org/10.1101/2021.03.23.434325


# Contact
* Hao Wu (haowu2@pennmedicine.upenn.edu) & Peng Hu (penghu41@gmail.com)
* [Wu lab](https://www.wulabupenn.org), Department of Genetics, University of Pennsylvania
